include/sensor.h

```diff
@@ -112,6 +112,13 @@ enum sensor_channel {
 	SENSOR_CHAN_BLUE,
 	/** Altitude, in meters */
 	SENSOR_CHAN_ALTITUDE,
+
+	/** PM1.0 in ug/m3 */
```

Comment:
Describe the channel in full words: 1.0 micro-metter Particulate Matter, in ug/m3

Do the same below